src/v2/Apps/FairOrginizer/helpers/getArticlesColumns.ts

```diff
@@ -0,0 +1,10 @@
+export function getArticlesColumns<T>(articles: T[]) {
+  const leftColumn: T[] = []
+  const rightColumn: T[] = []
+
+  articles.forEach((article: T, i) => {
+    ;(i % 2 === 0 ? leftColumn : rightColumn).push(article)
```

Comment:
**question** (non-blocking): Is the semicolon at the front of this line necessary? If so, I would like to learn more about it!

Reply:
That was auto formatted but let's see what I've found: _It is acceptable to begin the statement with a `;` to make sure that the statement is interpreted as a new one and not a continuation of the previous statement._
So as I understand it was put here because the array for pushing into is dinamically defined and that's not a regular syntax

Reply:
OK. It makes sense that it was auto-formatted. I still don't understand why the syntax to dynamically pick an array to push to needs this, but it seems harmless 🤷‍♂️ Thanks for the explanation!

Reply:
I think it's helpful in case developer forgot to close the previous line with `;` so that the new line might be interpreted as a continuation of it. If the code looked something like 
```
const value = someFn
(i % 2 === 0 ? leftColumn : rightColumn).push(article)
```
JS would probably consider it as 
`const value = someFn(i % 2 === 0 ? leftColumn : rightColumn).push(article)`

Reply:
That example is very clear 😅 Thank you!